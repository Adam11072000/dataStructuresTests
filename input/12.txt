init 244
addImage 81
mergeSuperPixels 81 172 194
quit