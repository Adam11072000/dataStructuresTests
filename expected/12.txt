init done.
getHighestScoredLabel: INVALID_INPUT
getHighestScoredLabel: INVALID_INPUT
addImage: SUCCESS
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
resetLabelScore: INVALID_INPUT
quit done.
